xray/plotting/plotting.py

```diff
@@ -221,6 +276,23 @@ def _plot2d(plotfunc):
         if None, use the default for the matplotlib function
     add_colorbar : Boolean, optional
         Adds colorbar to axis
+    vmin, vmax : floats, optional
+        Values to anchor the colormap, otherwise they are inferred from the
+        data and other keyword arguments. When a diverging dataset is inferred,
+        one of these values may be ignored.
+    cmap : matplotlib colormap name or object, optional
+        The mapping from data values to color space. If not provided, this
+        will be either be ``viridis`` (if the function infers a sequential
+        dataset) or ``RdBu_r`` (if the function infers a diverging dataset).
+    center : float, optional
+        The value at which to center the colormap. Passing this value implies
+        use of a diverging colormap.
+    robust : bool, optional
+        If True and ``vmin`` or ``vmax`` are absent, the colormap range is
+        computed with robust quantiles instead of the extreme values.
+    extend : {'neither', 'both', 'min', 'max'}, optional
```

Comment:
Was not immediately obvious what this does- this example is helpful- http://matplotlib.org/examples/pylab_examples/contourf_demo.html

Another possibility here would be to replace extend with
```
colorbar_kwargs : dict, optional
    Additional keyword arguments for pyplot.colorbar
```
and then pass this through to the the call to `plt.colorbar`. This is nice because it lets the user have complete control over the colorbar, and it stops the function signature from growing anymore (I sometimes find long function signatures intimidating)